tools/m2kcli/main.cpp

```diff
@@ -43,7 +44,7 @@ static const char *const helpMessage = "Usage:\n"
 				       "    analog-in           control the analogical input component\n"
 				       "    analog-out          control the analogical output component\n"
 				       "    digital             control the digital component\n"
-				       "    power-supply        control the power supply\n";
```

Comment:
Is this a mistake? Power supply should be removed?

Reply:
Yes, there was a mistake. It was modified in commit 12f433b.